src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp

```diff
@@ -27,7 +28,8 @@ namespace AppInstaller::CLI::Workflow
             }
             else
             {
-                if (context.Args.Contains(Execution::Args::Type::IgnoreLocalArchiveMalwareScan))
+                if (context.Args.Contains(Execution::Args::Type::IgnoreLocalArchiveMalwareScan) &&
+                    Settings::IsAdminSettingEnabled(Settings::AdminSetting::LocalArchiveMalwareScanOverride))
```

Comment:
>         Settings::IsAdminSettingEnabled(Settings::AdminSetting::LocalArchiveMalwareScanOverride)) [](http://example.com/codeflow?start=11&length=98)

Should there be a separate elseif block for when someone includes --ignore-local-archive-malware-scan but does not have the admin setting for LocalArchiveMalwareScanOverride enabled? If so, should there be a different error message specific to admin settings?

Reply:
This message is handled in command invocation.